tests/Unit/Helpers/DefinitionValidatorTest.php

```diff
@@ -32,8 +32,8 @@ public function testIntegerKeyOfArray(): void
     public function dataInvalidClass(): array
     {
         return [
-            [42, 'Invalid definition: invalid class name. Expected string, got int'],
-            ['', 'Invalid definition: empty class name.'],
+            [42, 'Invalid definition: class name must be a non-empty string, got "int".'],
+            ['', 'Invalid definition: class name must be a non-empty string, got "string".'],
```

Comment:
May be for empty string make unique message? Current message is confusing.

Reply:
I've fixed it [there](https://github.com/yiisoft/definitions/pull/57)

Reply:
Ok